src/main/java/com/iota/iri/model/persistables/Milestone.java

```diff
@@ -54,8 +54,18 @@ public void readMetadata(byte[] bytes) {
 
     }
 
+
     @Override
     public boolean merge() {
         return false;
     }
+    @Override
+    public Persistable mergeTwo(Persistable nrTwo){
+        return null;
```

Comment:
Can you change to ` throw new OperationNotSupportedException("This object is not mergeable");